helpdesk/views/public.py

```diff
@@ -106,7 +107,7 @@ def view_ticket(request):
     if ticket_req and email:
         queue, ticket_id = Ticket.queue_and_id_from_query(ticket_req)
         try:
-            ticket = Ticket.objects.get(id=ticket_id, submitter_email__iexact=email)
+            ticket = Ticket.objects.get(Q(id=ticket_id) & (Q(submitter_email__iexact=email) | Q(viewable_globally=True)))
```

Comment:
Beyond allowing the user to view global tickets like this, (which is obviously needed), do we also need to add a second list to the dashboard view? We currently have "Tickets submitted by you" (or similar) and it seems we would want to have "Global tickets" or so (I don't love that name yet) in a list above that.

Or is something like that done already and I missed it in reading the code?

Reply:
I agree with you Carl that the dashboard currently has a "Ticket submitted by you" list and a separate "Global tickets" list would make sense. I've made a note to have this as a ToDo for this ticket: https://trello.com/c/hS1XoPFs/2086-support-need-a-support-dashboard-view which I will rebase off this branch.